cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml

```diff
@@ -117,12 +117,12 @@ data:
     }
   {{- end }}
 
-  {{- if and (not .Values.agent) (not .Values.cloudAgent) (not (eq (include "aggregator.deployMethod" .) "disabled")) }}
+  {{- if and (not .Values.agent) (not .Values.cloudAgent) (not (eq (include "aggregator.deployMethod" .) "disabled")) (not .Values.kubecostAggregator.enabled) }}
     upstream aggregator {
         {{- if .Values.kubecostFrontend.useDefaultFqdn }}
         server {{ .Release.Name }}-aggregator.{{ .Release.Namespace }}.svc.cluster.local:9004;
         {{- else }}
-        {{- if (.Values.kubecostFrontend.aggregator).fqdn }}
+        {{- if and .Values.kubecostFrontend.aggregator .Values.kubecostFrontend.aggregator.fqdn }}
```

Comment:
I believe that the following two lines are equivalent? Did one not work for you?

```
{{- if (.Values.kubecostFrontend.aggregator).fqdn }}
```

```
{{- if and .Values.kubecostFrontend.aggregator .Values.kubecostFrontend.aggregator.fqdn }}
```